index.rst

```diff
@@ -522,6 +522,37 @@ cycle, technical managers work with the DM Project Manager and the Project
 Controls Specialist to ensure their plan for the cycle is well defined in both
 :term:`JIRA` and :term:`PMCS`.
 
+.. _sec-cycle-cadence:
+
+Cycle Cadence & Release Planning
+--------------------------------
+
+At the end of a cycle, the :term:`SQuaRE` team will coordinate a public
```

Comment:
Maybe we should be more specific and say "release manager"? (it is currently Frossie, and it is part of SQuaRE and it will stay in SQuaRE (no worries @frossie!) but it is more precise.